agents/tools/search_tools.py

```
@@ -0,0 +1,341 @@
+"""
+Aerospace-specific search tools for parts, chemicals, and tools research.
+"""
+
+from crewai.tools import BaseTool
+from typing import Dict, List, Any, Optional
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)